faster_whisper/transcribe.py

```diff
@@ -344,9 +348,11 @@ def load_vad_model(self, vad_onset=0.500, vad_offset=0.363, use_auth_token=None)
             "min_duration_on": 0.1,
             "min_duration_off": 0.1,
         }
+        # Separate vad_device from pipeline self.device
+        vad_device = "cuda" if torch.cuda.is_available() else "cpu"
 
         vad_pipeline = VoiceActivitySegmentation(
```

Comment:
Why can't we use the device from the pipeline? I thought it was a bug only. What does the pipeline do on GPU?

Reply:
Using CPU for VAD reduces the speed significantly for long files. We need GPU by default if available. Whereas we can keep CPU as the default for pipeline since the speed is similar with GPU pipeline.

Reply:
But this way you cannot change the device for VAD without changing the code. You can change device for the whisper model, for pipeline but not VAD.

Reply:
Yes, do you recommend putting a vad_device argument extra and keep it as cuda by default?

Reply:
I think it makes sense.